src/panels/lovelace/cards/hui-thermostat-card.ts

```diff
@@ -162,13 +167,19 @@ export class HuiThermostatCard extends LitElement implements LovelaceCard {
         width: 100%;
         font-size: 18px;
         line-height: 36px;
-        padding: 8px 30px 8px 30px;
+        padding: 8px;
```

Comment:
Should still keep padding to the right for the overflow:

![CleanShot 2023-12-18 at 15 53 10](https://github.com/home-assistant/frontend/assets/5662298/23ea7185-153a-4656-99fb-86cc9d3099a3)

Reply:
Oh, I missed to remove the overflow. It's replaced by the title.